src/app/components/leftbar/leftbar.component.html

```diff
@@ -1,18 +1,18 @@
 <div class="ccf-leftbar-wrapper">
   <div class="container">
     <div class="leftbar">
-      <div class="search">
-        <ccf-search-icon (click)="emitToggleEvent()"></ccf-search-icon>
+      <div [ngClass]="{'search': true, 'search-expanded': sidenavExpanded}">
```

Comment:
Prefer using [class.foo]="condition" when having just a couple of classes that you know the name of (i.e. [class.search-expanded]="sidenavExpanded")
Use class="c1 c2" for classes that are always present (i.e. the search class)